Dockerfile

```diff
@@ -42,10 +42,22 @@ FROM eclipse-temurin:17-jdk-focal as main
 
 RUN apt-get update && apt-get install -y libjemalloc-dev
 
+# Install Python and pip and clean up apt cache
+RUN apt-get update && apt-get install -y python3 python3-pip && rm -rf /var/lib/apt/lists/*
+
 WORKDIR /app
 
 COPY --from=build /app/pipelines/controller/target/controller-bundled.jar .
 
+COPY --from=build /app/pipelines/controller-cli ./controller-cli
+
+WORKDIR ./controller-cli
+
+# Install the controller-cli script
+RUN pip3 install .
```

Comment:
Have you tried running the script that is created with this? I mean using `docker exec`. The reason I am asking is because I think we don't have a self-contained script and it needs to have access to the dependencies; hence should be run from the "right" dir or with path env-vars set.

Reply:
Yeah I tested it on my local. I was able to successfully exec in and run the commands e.g `controller -h`

I think the reason it works is because we have the dependencies set up globally in the container and not a virtual environment local to the controller cli dir.